docs/UserGuide.md

```diff
@@ -77,156 +105,254 @@ Format: `help`
+Allows you to edit specific details of the applicant identified by the index number. The index number used corresponds to the one in the displayed applicant list. 
+Existing values will be overwritten by the input values
+
+<br>
 
 Format:
 ```
-edit i/INDEX [n/NAME] [p/PHONE] [e/EMAIL] [j/JOB] [r/ROUND] [s/SKILL]...
+edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [j/JOB] [r/ROUND] [s/SKILL]...
 ```
 
-* Only valid index are edited. e.g `5` will not edit if there are only `4` applicants in the list
-* Only positive index are edited. e.g `-1` will not edit the details of an applicant
+* Only valid indexes are edited. e.g If there are only `4` applicants in the app but `5` is specified, then the intended action will not be carried out.
+* Only positive indexes are edited. e.g As we label our applicants starting from `1...`, an index of `-1` will not be tagged to an applicant.
+
+<br>
 
 Example:
 ```
 edit 1 p/91234567 e/johndoe@example.com
 ```
 Sample Output:
 ```
-Edited Applicant: David Lee; Phone: 91234567; Email: johndoe@example.com; Job: Risk Assessment Associate; Round: Stock Pitch Assessment; Skills: [Accounting][Equities][Cryptocurrency]
+Edited Applicant: David Lee; 
+Phone: 91234567; 
+Email: johndoe@example.com; 
+Job: Risk Assessment Associate; 
+Round: Stock Pitch Assessment; 
+Skills: [Accounting][Equities][Cryptocurrency]
+```
+
+[Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
+
+---
+### Flagging an applicant : `flag`
+
+Allows you to flag an applicant identified by the index number. The index number used corresponds to the one in the displayed applicant list. Flagged applicants will appear at the top of the list and are identified with a flag symbol.
+
+<br>
+
+Format:
+```
+flag INDEX
+```
+
+* Only valid indexes are flagged. e.g If there are only `4` applicants in the app but `5` is specified, then the intended action will not be carried out.
+* Only positive indexes are flagged. e.g As we label our applicants starting from `1...`, an index of `-1` will not be tagged to an applicant.
+* Flag acts like a toggle. To un-flag the applicant, you may simply re-type the same command.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Try flagging the first applicant by typing `flag 1`. You should see it at the top with a flag symbol. Then try un-flagging the same applicant with `flag 1`.
+</div>
+
+<br>
+
+Example:
+```
+flag 1
+```
+Sample Output:
+```
+Flagged Applicant: David Lee; 
+Phone: 91234567; 
+Email: johndoe@example.com; 
+Job: Risk Assessment Associate; 
+Round: Stock Pitch Assessment; 
+Skills: [Accounting][Equities][Cryptocurrency]
 ```
 
 [Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
 
 ---
 ### Deleting an applicant : `delete`
 
-Deletes a specific job applicant, specified by index.
+Allows you to delete a specific job applicant, specified by index.
```

Comment:
seems like 'applicant' is used in other features, probably better to standardize it?
```suggestion
Allows you to delete a specific applicant, specified by index.
```

Reply:
Fixed!